### PR TITLE
Add support for JP 6.0DP

### PR DESCRIPTION
**Pull request by qqice**
solved #470

### CODE REVIEW

File: jtop/core/memory.py [hunk 1]
```diff
@@ -425,7 +425,7 @@ def get_status(self, mem_total):
             memory['EMC']['online'] = True
             # Percentage utilization
             # https://forums.developer.nvidia.com/t/real-time-emc-bandwidth-with-sysfs/107479/3
-            utilization = int(cat(self._root_path + "/actmon_avg_activity/mc_all"))
+            utilization = int(cat(self._root_path + "/debug/cactmon/mc_all"))
```
Comment (jasl) [inline jtop/core/memory.py]: Maybe you can detect if the file exists first and fall back to the old path if it is not found.
I think a better way is to detect paths on the service booting

Comment (qqice) [reply jtop/core/memory.py]: Thanks for your suggestion. Another commit was added to address the issue.

**Comment by rbonghi** [comment]
Thank you so much for this pull request, soon as possible I'll test, merge and publish a new release

**Comment by CharlesMod** [comment]
Hi there! Any updates on a new release? Currently running 6.0 and jtop is missing from my life haha

**Comment by rbonghi** [comment]
Promise after Christmas I'll merge all PR

**Comment by rbonghi** [comment]
Work in progress 

image

**Comment by rbonghi** [comment]
The first step done; I noticed other things to fix before releasing the new version.

**Comment by jasl** [comment]
> The first step done; I noticed other things to fix before releasing the new version.

Thank you for the gift! Merry Christmas

**Comment by lars-nagel** [comment]
Happy new year! I hope you @rbonghi don't have too much trouble fixing the other things :)

**Comment by rbonghi** [comment]
Hi @lars-nagel
I reported my status update for this release https://github.com/rbonghi/jetson_stats/issues/470 
There is an issue with JP6 to detect the Jetson model. I'm looking for a workaround

**Pull request merged**